index.html

```diff
@@ -1,19 +1,74 @@
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="stylesheet" href="store.html">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header>
+            <nav>
+                <img class="logo" src="img/karma-logo.svg" alt="logo">
+                <a href="" class="a-options">Login</a>
+                <a href="" class="a-options">Help</a>
+                <a href="" class="a-options">Blog</a>
+                <a href="store.html" class="a-options">Store</a>
+                <a href="" class="a-options">How it Works</a>
+                <a href="index.html" class="a-options"><strong>Meet Karma</strong></a>
+            </nav>
+        </header>
+        <main>
+            <div class="container">
+                <img class="bg-image" src="img/first-background.jpg" alt="bg-img">
+                <div class="bg-text">
+                    <p class="p-1">Introducing Karma</p>
+                    <p class="p-2">Bring WiFi with you, everywhere you go.<p>
+                    <button class="btn-learn-more">Learn More</button>
+                </div>
+            </div>
+            <div>
+                <p class="p-3">Everyone needs a little Karma.</p>
+            </div>
+            <div class="image-1">
+                <div>
+                    <img class="img-1" src="img/icon-devices.svg" alt="device-img">
+                    <p class="p-4">Internet for all devices</p>
+                </div>
+                <div>
+                    <img class="img-1" src="img/icon-coffee.svg" alt="coffee-img">
+                    <p class="p-4">Boost your productivity</p>
+                </div>
+                <div>
+                    <img class="img-1" src="img/icon-refill.svg" alt="refill-img">
+                    <p class="p-4" >Pay as You Go</p>
+                </div>
+            </div>
+            <aside class="addition-1">
+                <div class="aside-1">
+                    <p class="p-new"> Wherever I am, I just don´t worry about my connection anymore! </p>
+                </div>
+                <div class="aside-2">
+                    <a href="store.html" class="get-karma-today">Get Karma today</a>
```

Comment:
Also, making the button text bolder would make it much eaiser to read. Using font-weight